iso15118/secc/states/iso15118_2_states.py

```diff
@@ -930,9 +1009,17 @@ async def process_message(
         # self.comm_session.contract_cert_chain attribute.
         auth_status: EVSEProcessing = EVSEProcessing.ONGOING
         next_state: Type["State"] = Authorization
-        if await self.comm_session.evse_controller.is_authorized() == (
-            AuthorizationStatus.ACCEPTED
-        ):
+        # TODO: how should we obtain/hold the token for EIM?  Other cases?
+        id_token = (
+            self.comm_session.emaid
+            if self.comm_session.selected_auth_option == AuthEnum.PNC_V2
+            else None
+        )
+        authorization_result = await self.comm_session.evse_controller.is_authorized(
+            id_token=id_token,
+            id_token_type=self.comm_session.selected_auth_option,
```

Comment:
just again a note about the auth option may not be what we want

Reply:
As mentioned, have resolved this.